recipes/database.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Cookbook Name:: timesync
+# Recipe:: database
+#
+# Copyright 2015 Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+bash 'database' do
+  user 'root'
+  cwd "#{node['timesync']['application_path']}/source"
+  code <<-EOH
+    npm run migrations
+    chown #{node['timesync']['user']}:#{node['timesync']['group']} dev.sqlite3
```

Comment:
this is a temporary measure -- we have plans in place to move to postgres, which will be an entire new can of worms. Mostly the reason this is done as root is because the dir is not writable by ``timesync``, and I didn't want to change dir permissions so the migrations could be run and then change it back.

I can do that if you would prefer.